keyboards/handwired/plonk/rules.mk

```diff
@@ -0,0 +1,46 @@
+# MCU name
+# MCU = atmega32u4
+MCU = STM32F303
+# CTPC = yes
+ALLOW_WARNINGS = yes
```

Comment:
Why this?  

It looks like it compiles fine. Also any warnings may be important, too.

Reply:
Ah, I had forgotten about that. I have a habit of leaving old code in, just commented out for when I inevitably screw something up. I will fix that. As for allowing warnings, Arch is using a compiler that is, a little broken right now. I'll remove that argument and just add it in at compile time.